source/OutfitInfoDisplay.cpp

```diff
@@ -406,6 +407,7 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	vector<double> values = {
 		outfit.ShieldDamage(),
 		outfit.HullDamage(),
+		outfit.AsteroidDamage(),
```

Comment:
We don't need every weapon in the game now displaying an asteroid damage, so if the asteroid damage matches the hull damage (i.e. asteroid damage fell back onto hull damage because it was not specified for the weapon) don't show it (i.e. give it a value of 0 for the display).
```suggestion
		outfit.AsteroidDamage() != outfit.HullDamage() ? outfit.AsteroidDamage() : 0.
```

Reply:
Doesn't that conditional statement need a comma after it or is it fine with a period? Just checking as I don't know c++ that well.

Reply:
It does. My bad. Updated the suggestion.